doc/tutorial/basic/tutorial.rst

```diff
@@ -136,11 +136,30 @@ learn::
     <sphx_glr_auto_examples_classification_plot_digits_classification.py>` illustrates how starting
     from the original problem one can shape the data for consumption in
     scikit-learn.
-    
+
 .. topic:: Loading from external datasets
 
     To load from an external dataset, please refer to :ref:`loading external datasets <external_datasets>`.
 
+.. topic:: Accessing source files
+
+    All standard datasets which you can import with ``load_`` have underlying source files that
+    you can read manually (consider numpy.loadtxt and pandas for analysis).
+    The data and target can be stored in one file (e.g. iris, boston, breast_cancer) or
+    in several (e.g. diabetes, linnerud).
+
+      >>> from sklearn.datasets import load_boston
+      >>> boston = load_boston()
+      >>> print(boston.filename)  # doctest: +SKIP
+      (some-path)/sklearn/datasets/data/boston_house_prices.csv
```

Comment:
I tried this approach first:
```python 
-      >>> print(diabetes.target_filename)  # doctest: +SKIP
-      (some-path)/sklearn/datasets/data/diabetes_target.csv.gz
+      >>> print(diabetes.target_filename)  # doctest: +ELLIPSIS
+      ...sklearn/datasets/data/diabetes_target.csv.gz
```

```python
ValueError: line 161 of the docstring for tutorial.rst lacks blank after ...: '      ...sklearn/datasets/data/diabetes_target.csv.gz'
```

and with:
```python
...
+      ...sklearn/datasets/data/diabetes_target.csv.gz
```

```python
File ".../sklearn_dev/doc/tutorial/basic/tutorial.rst", line 160, in tutorial.rst
Failed example:
    print(diabetes.target_filename)  # doctest: +ELLIPSIS
    sklearn/datasets/data/diabetes_target.csv.gz
Expected nothing
Got:
    .../sklearn_dev/sklearn/datasets/data/diabetes_target.csv.gz

>>  raise self.failureException(self.format_failure(<StringIO.StringIO instance at 0x10e4ed5f0>.getvalue()))
```

That is why I end up with "+SKIP"